galpy/potential_src/Potential.py

```diff
@@ -1619,55 +1636,59 @@ def ttensor(self,R,z,phi):
             
         INPUT:
         
-            Pot - Potential instance or list of such instances
-            
             R - Galactocentric radius (can be Quantity)
             
             z - height (can be Quantity)
             
+            phi - azimuth (optional; can be Quantity)
+            
+            t - time (optional; can be Quantity)
+            
+            eigenval - return eigenvalues if true (optional; boolean)
             
         OUTPUT:
         
             Tidal Tensor
         
         HISTORY:
         
-            2018-03-07 - Written - Webb (UofT)
-            
+            2018-03-21 - Written - Webb (UofT)
+
         """
         if self.isNonAxi:
             #Make sure potential is axisymmetric
-            print("Error - cannot calculate tidal tensor for non-axisymmetric potential")
+            raise PotentialError("Cannot calculate ttensor for non-axisymmetric potential")
```

Comment:
So again everything is just computed based on local forces and second derivatives, so why can this not be done for a non-axisymmetric potential?